test/index.js

```diff
@@ -7,7 +7,8 @@ describe('Marked renderer', () => {
   const ctx = {
     config: {
       marked: {}
-    }
+    },
+    execFilterSync: function() {}
```

Comment:
Add `hexojs/hexo` to `devDependencies`, and you could start a Hexo instance during the unit test:

https://github.com/hexojs/hexo-generator-archive/blob/master/test/index.js#L4-L7